src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -45,6 +53,25 @@ public DynamicallyAccessedMemberTypes GetParameterAnnotation (MethodDefinition m
 			return DynamicallyAccessedMemberTypes.None;
 		}
 
+		private DynamicallyAccessedMemberTypes GetMemberTypesForDynamicallyAccessedMemberAttribute (ICustomAttributeProvider provider)
+		{
+			if (_source.HasCustomAttributes (provider)) {
+				foreach (var attribute in _source.GetCustomAttributes (provider)) {
+					if (IsDynamicallyAccessedMembersAttribute (attribute)) {
+						if (attribute.ConstructorArguments.Count == 0) {
+							_context.LogMessage (MessageContainer.CreateWarningMessage ($"DynamicallyAccessedMembers attribute was specified but no argument was proportioned", 2020));
+						} else if (attribute.ConstructorArguments.Count == 1) {
+							var arguments = attribute.ConstructorArguments.ToArray ();
+							return (DynamicallyAccessedMemberTypes) arguments[0].Value;
```

Comment:
You cannot cast external value to local type

Reply:
Same comment as in the previous PR - this is an enum type, this is basically the only case where it's "safe" since it's almost like a primitive type.

Reply:
I don't think that's the case. If someone declares it's own type (we matching on names) then you cannot cast the enum type to "runtime" version of the type as they have different assembly identity.

Reply:
You're right - we have to go through integer:
```C#
return (DynamicallyAccessedMemberTypes)(int)arguments[0].Value;
```

Then it should be correct - as we will simply map the number to the linker understanding of the flags.